app/views/buyers/invitations/index.html.erb

```diff
@@ -4,6 +4,12 @@
 
 <% content_for :page_header_title, "Sent invitations for #{@account.org_name}" %>
 
+<% content_for :javascripts do %>
+  <%# HACK: temporary reuse css of existing pack "table_toolbar". Once toolbar is implemented,
+      import the whole pack with "javascript_packs_with_chunks_tag" %>
+  <%= stylesheet_packs_chunks_tag 'table_toolbar' %>
```

Comment:
I'm confused. Why can't you just call `<%= stylesheet_packs_chunks_tag 'table_toolbar' %>`?, why pass it to `content_for :javascripts`?

Reply:
You can of course. But it's very hard to debug (and maintain) an HTML with scripts and stylesheets all over the body. By passing it to `:javascripts` we ensure they are added at the bottom inside the `<head>`.

Reply:
I understand. And couldn't it be better to add a `yield :stylesheets` in the layout and use `content_for :stylesheets` ?

Reply:
Yes and no. There is already a `:stylesheets` section near the top of the `<head>`, https://github.com/3scale/porta/blob/25fa9bc53bdd3eeb8f2a9c8e882f771c85c3c3cb/app/views/provider/_theme.html.erb#L3

But adding styles there would break everything again, because of the import order. The javascript packs include styles as well and they are added together. By appending stylesheets to `:javascripts` we guarantee the import order. It's just a naming issue, we could call it `:packs` instead of `:javascripts`. Or maybe there is a better way to separate script and stylesheet and add them to `:javascripts` and `:stylesheets` but in any case I think it's a improvement to be made in a different PR.

Reply:
> but in any case I think it's a improvement to be made in a different PR.

Agreed. Thanks for the explanation.